docs/pages/versions/unversioned/sdk/google.md

```diff
@@ -16,7 +16,7 @@ You'll get an access token after a successful login. Once you have the token, if
 
 ## Installation
 
-For [managed](../../introduction/managed-vs-bare/#managed-workflow) apps, you'll need to run `expo install expo-google-app-auth`. To use it in a [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native app, you will need to run `npx pod-install` and do a new build after installing the package because this library pulls in [**`expo-app-auth`**][expo-app-auth] as a dependency.
+For [managed](../../introduction/managed-vs-bare/#managed-workflow) apps, you'll need to run `expo install expo-google-app-auth`. To use it in a [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native app, you will need to run `npx pod-install` and do a new build after installing the package because this library pulls in [**`expo-app-auth`**](https://github.com/expo/expo/blob/master/docs/pages/versions/unversioned/sdk/app-auth.md) as a dependency.
```

Comment:
```suggestion
For [managed](../../introduction/managed-vs-bare/#managed-workflow) apps, you'll need to run `expo install expo-google-app-auth`. To use it in a [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native app, you will need to run `npx pod-install` and do a new build after installing the package because this library pulls in [**`expo-app-auth`**](/versions/latest/sdk/app-auth/) as a dependency.
```

Reply:
can you copy this change over to https://github.com/expo/expo/blob/master/docs/pages/versions/v39.0.0/sdk/google.md also?

Reply:
sure! here you are
btw nice talk at the "https://www.react-native.eu/" I really enjoyed it!